node/src/Commands.ts

```diff
@@ -4141,3 +4141,55 @@ export function createScriptFlush(mode?: FlushMode): command_request.Command {
 export function createScriptKill(): command_request.Command {
     return createCommand(RequestType.ScriptKill, []);
 }
+
+/** Represents options for formatting JSON data, to be used in  the [JSON.GET](https://redis.io/commands/json.get/) command.
+    Args:
+        paths
+        indent: Sets an indentation string for nested levels. Defaults to None.
+        newline: Sets a string that's printed at the end of each line. Defaults to None.
+        space: Sets a string that's put between a key and a value. Defaults to None. */
+export type JsonGetOptions = {
```

Comment:
I am pretty sure we document the type differently and we use interface

(I know this si a draft so is its something youre aware of ignore)